### PR TITLE
env variable is added to auth

### PR DESCRIPTION
**Pull request by Sushanth-Hebri**
Auth key is added with env variable

### CODE REVIEW

**Comment by Sushanth-Hebri** [comment]
Environment variable used "REACT_APP_API_KEY" ,
add .env file to .gitignore

File: .env
Comment (Avdhesh-Varshney) [inline .env:1]: @Sushanth-Hebri still this key will be visible. So, you have to delete this file. I will add this key in deployement env variables. and resolve the conflicts.

File: src/components/Quotes/QuotesSection.jsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ const QuoteSection = () => {
     try {
       const response = await fetch('https://api.yourquoteapi.com/quotes?author=Chanakya', {
         headers: {
-          'Authorization': `4fe303f4bamshb97ef5b1dd575e0p1ccc51jsnd9644d3c587b`
+          'Authorization': process.env.REACT_APP_API_KEY
```
Comment (Avdhesh-Varshney) [inline src/components/Quotes/QuotesSection.jsx:16]: named it `QUOTES_API_KEY`

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

**Comment by Sushanth-Hebri** [comment]
solved this in #124 , auth key must start with REACT_APP for safety , for custom key there need to be separate wired config ,
now auth key is REACT_APP_QUOTES_API_KEY

**Pull request closed without merging**